promptulate/tools/shell/api_wrapper.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+from io import StringIO
+from typing import Dict, Optional
+from pydantic import BaseModel, Field
+
+
+class ShellAPIWrapper(BaseModel):
```

Comment:
BaseModel seems useless. I think removing BaseModel can achieve the same result.

Reply:
ok